src/backend/expungeservice/expunger.py

```diff
@@ -59,20 +62,27 @@ def run(self) -> bool:
 
             if charge.disposition.status == DispositionStatus.NO_COMPLAINT:
                 eligibility_dates.append(
-                    (charge.disposition.date + relativedelta(years=1), "Time-ineligible under 137.225(1)(b)")
+                    (
+                        charge.disposition.date + relativedelta(years=1),
+                        "One year from date of no-complaint arrest (137.225(1)(b))",
+                    )
                 )
 
             if most_recent_blocking_conviction:
+                other = "other " if charge.convicted() else ""
```

Comment:
```suggestion
                conviction_string = "other conviction" if charge.convicted() else "conviction"
```

Fairly nitpicky but I did spend 15 seconds feeling aesthetically annoyed that I'm guessing some future person might too.

Reply:
ha fair. I did imagine mine my rustle someone's jimmies